### PR TITLE
Flask project setup

### PR DESCRIPTION
**Pull request by JakeDarling**
Let me know if I'm missing anything

### CODE REVIEW

**Review by Matthew-Crawford — APPROVED** [review]

**Comment by JakeDarling** [comment]
README needs updating

File: knockserver/database.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+import config
+import os
+
+# Get the database string from config
+db_connect_string = config.DATABASE_URI
```
Comment (ip2176) [inline knockserver/database.py:8]: Do we need this extra line?  It looks like it only used once, and you can just substitute it with config.DATABASE_URI one line 10.

**Review by ip2176 — COMMENTED** [review]
Looks good except for the one line I commented on in database.py, line 8.  It doesn't look like we need that and we can just reference the DB URI directly right?

File: knockserver/database.py (hunk 1 above)
Comment (corbanmailloux) [reply knockserver/database.py:8]: I like this variable defined because it extracts the `config` calls to the top.

**Review by corbanmailloux — APPROVED** [review]
I like it. I didn't test if the DB connection functions, but this seems like a good starting point for our project. 👍

**Review by TempusShift — APPROVED** [review]

**Pull request merged**